vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionAboutPageManager.java

```diff
@@ -41,4 +41,14 @@ public ExhibitionAboutPage store(ExhibitionAboutPage exhibitionAboutPage) {
         return repo.save(exhibitionAboutPage);
     }
     
+    /**
+     *  This method returns the Exhibition About Page set by a Staff, if empty
```

Comment:
This is actually not true. It returns the first page it finds.

Reply:
Changed